lcm/dbg.h

```diff
@@ -5,97 +5,75 @@
 extern "C" {
 #endif
 
+#include <stdarg.h>
 #include <stdio.h>
-#include <string.h>
 #include <stdlib.h>
-#include <stdarg.h>
+#include <string.h>
 
-#define _NORMAL_    "\x1b[0m"
-#define _BLACK_     "\x1b[30;47m"
-#define _RED_       "\x1b[31;40m"
-#define _GREEN_     "\x1b[32;40m"
-#define _YELLOW_    "\x1b[33;40m"
-#define _BLUE_      "\x1b[34;40m"
-#define _MAGENTA_   "\x1b[35;40m"
-#define _CYAN_      "\x1b[36;40m"
-#define _WHITE_     "\x1b[37;40m"
-
-#define _BRED_      "\x1b[1;31;40m"
-#define _BGREEN_    "\x1b[1;32;40m"
-#define _BYELLOW_   "\x1b[1;33;40m"
-#define _BBLUE_     "\x1b[1;34;40m"
-#define _BMAGENTA_  "\x1b[1;35;40m"
-#define _BCYAN_     "\x1b[1;36;40m"
-#define _BWHITE_    "\x1b[1;37;40m"
+#define _NORMAL_ "\x1b[0m"
+#define _BLACK_ "\x1b[30;47m"
+#define _RED_ "\x1b[31;40m"
+#define _GREEN_ "\x1b[32;40m"
+#define _YELLOW_ "\x1b[33;40m"
+#define _BLUE_ "\x1b[34;40m"
+#define _MAGENTA_ "\x1b[35;40m"
+#define _CYAN_ "\x1b[36;40m"
+#define _WHITE_ "\x1b[37;40m"
+
+#define _BRED_ "\x1b[1;31;40m"
+#define _BGREEN_ "\x1b[1;32;40m"
+#define _BYELLOW_ "\x1b[1;33;40m"
+#define _BBLUE_ "\x1b[1;34;40m"
+#define _BMAGENTA_ "\x1b[1;35;40m"
+#define _BCYAN_ "\x1b[1;36;40m"
+#define _BWHITE_ "\x1b[1;37;40m"
 
 #define DBG_MODE(x) (1ULL << (x))
 
-#define DBG_ALL     (~0ULL)
-#define DBG_ERROR   DBG_MODE(0)
+#define DBG_ALL (~0ULL)
+#define DBG_ERROR DBG_MODE(0)
 #define DBG_DEFAULT DBG_ERROR
 
 // ================== add debugging modes here ======================
 
-#define DBG_TEST    DBG_MODE(1) /* foo */
-#define DBG_LCM     DBG_MODE(2)
+#define DBG_TEST DBG_MODE(1) /* foo */
+#define DBG_LCM DBG_MODE(2)
 #define DBG_LCM_MSG DBG_MODE(3)
-#define DBG_MAIN    DBG_MODE(4)
-#define DBG_PYTHON  DBG_MODE(5)
-#define DBG_6       DBG_MODE(6)
-#define DBG_7       DBG_MODE(7)
-#define DBG_8       DBG_MODE(8)
-#define DBG_9       DBG_MODE(9)
-#define DBG_10      DBG_MODE(10)
-#define DBG_11      DBG_MODE(11)
-#define DBG_12      DBG_MODE(12)
-#define DBG_13      DBG_MODE(13)
-#define DBG_14      DBG_MODE(14)
-#define DBG_15      DBG_MODE(15)
-#define DBG_16      DBG_MODE(16)
+#define DBG_MAIN DBG_MODE(4)
+#define DBG_PYTHON DBG_MODE(5)
+#define DBG_6 DBG_MODE(6)
+#define DBG_7 DBG_MODE(7)
+#define DBG_8 DBG_MODE(8)
+#define DBG_9 DBG_MODE(9)
+#define DBG_10 DBG_MODE(10)
+#define DBG_11 DBG_MODE(11)
+#define DBG_12 DBG_MODE(12)
+#define DBG_13 DBG_MODE(13)
+#define DBG_14 DBG_MODE(14)
+#define DBG_15 DBG_MODE(15)
+#define DBG_16 DBG_MODE(16)
 
 /// There can be no white space in these strings
 
-#define DBG_NAMETAB \
-{ "all", DBG_ALL }, \
```

Comment:
This is even more an anti-improvement.

Reply:
Yeah, let me take a closer look at all macros. These are good candidates for disabling formatting around them.